src/app/register/register.component.ts

```diff
@@ -63,6 +70,23 @@ export class RegisterComponent implements OnInit, OnDestroy {
     });
   }
 
+  ngAfterViewInit() {
+    if (! flags.friendlyCaptchaEnabled) {
+      return;
+    }
+    const widget = new WidgetInstance(this.friendlyCaptcha.nativeElement, {
```

Comment:
I think this may run in a server side render too, and doesn't need to?

Reply:
Agreed - will change.

Reply:
Fixed in 55c5f374